packages/snap-preact-components/src/components/Atoms/SearchHeader/SearchHeader.tsx

```diff
@@ -104,13 +108,24 @@ export const SearchHeader = observer((properties: SearchHeaderProps): JSX.Elemen
 							</>
 						) : (
 							pagination?.totalResults === 0 && (
-								<h3
-									className={classNames('ss__search-header__title', 'ss__search-header__title--no-results')}
-									aria-atomic="true"
-									aria-live="polite"
-									aria-label={`No results found for ${search?.query?.string}`}
-									dangerouslySetInnerHTML={{ __html: noResultsText as string }}
-								></h3>
+								<div className="ss__search-header__no-results-wrapper">
+									<h3
+										className={classNames('ss__search-header__title', 'ss__search-header__title--no-results')}
+										aria-atomic="true"
+										aria-live="polite"
+										aria-label={`No results found for ${search?.query?.string}`}
+										dangerouslySetInnerHTML={{ __html: noResultsText as string }}
+									></h3>
+
+									{search?.didYouMean && (
+										<h3
+											className="ss__search-header__title--no-results__dym"
```

Comment:
All of the others are using the classname `ss__search-header__title` with the modifier class. Perhaps we should keep it consistent with this one as well?